config.yaml

```diff
@@ -449,6 +453,41 @@ mqtt:
     dawn_dusk_topic: 'insteon/{{address}}/dawn'
     dawn_dusk_payload: '{{is_dawn_str.upper()}}'
 
+  #------------------------------------------------------------------------
+  # Hidden Door Sensors
+  #------------------------------------------------------------------------
+
+  # Hidden Door Sensors will use the state and low battery configuration
+  # inputs from battery_sensor and add battery voltage, low battery 
+  # threshold and heart beat interval which is configured here.
+  #
+  # To register the hidden door sensor in Home Assistant use MQTT binary
+  # sensor with a configuration like:
+  #   binary_sensor:
+  #     - platform: mqtt
+  #TODO       state_topic: 'insteon/aa.bb.cc/dawn'
+  #TODO       device_class: 'light'
+  hidden_door:
+    # Output topic and payload.  This message is sent
+    # whenever the device is polled and a new voltage, low battery voltage
+    # or heart beat interval is obtained from the device.
+    # Available variables for templating are:
+    #   address = 'aa.bb.cc'
+    #   name = 'device name'
+    #   batt_volt = raw insteon voltage level
+    #   low_batt_volt = raw insteon voltage level to trigger low bat
+    #   hb_interval = interval between heart beats sent by device.
+    #                 Interval = 5 mins x this setting.  0 = 24 hours
+  
+    battery_voltage_topic: 'insteon/{{address}}/battery/state'
+    battery_voltage_payload: '{"voltage" : {{batt_volt}}}'
+
+    low_battery_voltage_topic: 'insteon/{{address}}/low_battery/state'
+    low_battery_voltage_payload: '{"low_voltage_threshold" : {{low_batt_volt}}}'
+
+    heart_beat_interval_topic: 'insteon/{{address}}/heartbeat/state'
+    heart_beat_interval_payload: '{"heart_beat_interval" : {{hb_interval}}}'
+
```

Comment:
I realize a user can change theses, but few users understand templates, and even fewer read the config file.  It would be better if these topics in their default state didn't deviate from what other devices use.  
1. There shouldn't be a "/state" on the battery topic.
2. I think the "battery" topic should work as much like it does for other devices, that is in it's default state emitting 'ON' if the battery is low and not the voltage.  So the payload would be '{{is_low_str.upper()}}'.  The additional voltage variable is nice and users can alter their config if they want it.
3. Is the 'low battery' topic an input?  Is that needed, are users likely to change their low battery voltage amount programatically?  I think this is better as a `set-flag` option unless you can convince me otherwise.  Same thing for the heartbeat.

Reply:
Oh, I see the battery/state is an additional topic.  I still think this is probably unnecessary and adding too many topics, but if you want to add it, I could go with `insteon/{{address}}/battery_voltage`

Reply:
1 & 2. OK, if you agree, I'll change these three data items to:
`insteon/{{address}}/battery_voltage`
`insteon/{{address}}/low_battery_voltage`
`insteon/{{address}}/heart_beat_interval`

I'm also open to just removing the low bat level and heart beat interval (see explanation below in 3), let me know.

3. When you send the get extended status message to the HDS, you get three data items:  Current battery voltage (raw level), Battery low level threshold (raw level), and heart beat interval (raw level).  So, since we attempt go get the battery level form the device anyway, these other two items come free.  While I suspect they wouldn't be used often, I figured why not report them over MQTT for those who want them.  So to answer your question, no this is not an input channel, this is output only.

Reply:
Got it.  I would just drop them as topics then.  It seems to me these values should just be output in the UI response to the get_extended status message.  Output, MQTT topics should be things we anticipate users building automation around.

Again, just trying to keep things consistent and clean for the average user.

Reply:
Sounds good - I'll do that.

Reply:
complete